src/bci_build/package/php.py

```diff
@@ -138,10 +141,14 @@ def _create_php_bci(
         pretty_name=f"{str(php_variant)} {php_version}",
         package_name=f"{str(php_variant).lower()}{php_version}-image",
         os_version=os_version,
-        is_latest=php_version == _LATEST_PHP_VERSION
-        and os_version in CAN_BE_LATEST_OS_VERSION,
+        is_latest=(
+            php_version == _LATEST_PHP_VERSION
+            and os_version in CAN_BE_LATEST_OS_VERSION
+        ),
         package_list=[
             f"php{php_version}",
+            # required for the interactive shell to work
+            f"php{php_version}-readline",
```

Comment:
can  you please update or remove https://github.com/SUSE/BCI-dockerfile-generator/pull/1156/files#diff-67354e8f15316af72a6752f09056dab59ecc4a47f1c11d0e4afbf6e7a1e70fe4R128-R132 then?

Reply:
Good catch, thanks!